src/script/components/list/ConversationListCell.tsx

```diff
@@ -119,7 +110,7 @@ const ConversationListCell: React.FC<ConversationListCellProps> = ({
   };
 
   return (
-    <li ref={setViewportElementRef}>
+    <li onContextMenu={openContextMenu}>
```

Comment:
No idea why this was made this way initially, but new way is the react way of things :D